File: Serene/Serene.Script/Imports/ServiceContracts/Northwind.OrderShippingState.cs

namespace Serene.Northwind
{
    using Serenity;
    using Serenity.ComponentModel;
    using System;
    using System.Collections;
    using System.Collections.Generic;
    using System.ComponentModel;
    using System.Runtime.CompilerServices;

    [EnumKey("Northwind.OrderShippingState"), PreserveMemberCase]
    public enum OrderShippingState
    {
        NotShipped = 0,
        Shipped = 1
    }
    
}

File: Serene/Serene.Web/Modules/Imports/ServerTypings/Northwind.OrderShippingState.ts
namespace Serene.Northwind {
    export enum OrderShippingState {
        NotShipped = 0,
        Shipped = 1
    }
    Serenity.Decorators.addAttribute(OrderShippingState, new Serenity.EnumKeyAttribute('Northwind.OrderShippingState'));
}

